packages/cli-main/src/index.ts

```diff
@@ -1,9 +1,10 @@
+import {constants} from '@shopify/cli-kit'
 import {runCLI} from '@shopify/cli-kit/node/cli'
 
 async function runShopifyCLI() {
   await runCLI({
     moduleURL: import.meta.url,
-    logFilename: 'shopify.cli.log',
+    logFilename: constants.logStreams.cli,
```

Comment:
What are the use cases for using different log files internally?

Reply:
Honestly, I've been questioning it myself. I originally implemented this way because it seemed like a good idea. But honestly it's probably overkill, and we can build in other ways of distinguishing the logs. Some of those ways are in this PR already.